samples/springboot3/alt-pet-store/pom.xml

```diff
@@ -31,13 +31,13 @@
     <dependencies>
         <dependency>
             <groupId>org.springframework.boot</groupId>
-            <artifactId>spring-boot-starter-web</artifactId>
```

Comment:
spring-web should be present somehow as the Controller uses the annotations.

Reply:
spring-web is brought transitively via serverless-java-container and s-c-functions. I just tested it, rebuilt, redeployed and all is working. Let me know if you see otherwise

Reply:
true